xbot_service_interface/README.md

```
@@ -82,8 +99,94 @@ val = echo.registers['Prefix']
+`xbot-shell` is an IPython-based REPL for exploring and testing services live.
+Requires the `[shell]` extras (`ipython`, `rich`).
+
+```bash
+xbot-shell               # bind all interfaces
+xbot-shell --bind 192.168.1.x   # specific interface
+```
+
+On startup the shell listens for service advertisements on the multicast group
+and prints each service as it appears:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)